plugins/experimental/uri_signing/jwt.c

```diff
@@ -77,6 +77,9 @@ jwt_delete(struct jwt *jwt)
   if (!jwt) {
     return;
   }
+  if (jwt->aud) {
```

Comment:
For consistency's sake the other decref calls rely on the json_decref function to do the decref checking.